src/components/vm/consoles/VncConsole.tsx

```diff
@@ -112,13 +139,15 @@ export const VncConsole = ({
             repeaterID,
             shared,
         };
-        rfb.current = new RFB(novncElem.current, url, options);
+        rfb.current = new RFB_constructor(novncElem.current!, url, options);
         addEventListeners();
-        rfb.current.viewOnly = viewOnly;
-        rfb.current.clipViewport = clipViewport;
-        rfb.current.dragViewport = dragViewport;
-        rfb.current.scaleViewport = scaleViewport;
-        rfb.current.resizeSession = resizeSession;
+        if (rfb.current) {
```

Comment:
This is probably better written as

```
const rfb_object = new RFB_constructor(...);
rfb_object.viewOnly = ...;
rfb.current= rfb_object;
```

to avoid the bogus conditional.